algopy_testing/tests/conftest.py

```diff
@@ -22,3 +28,25 @@ def algod_client() -> AlgodClient:
 @pytest.fixture(scope="session")
 def indexer_client() -> IndexerClient:
     return get_indexer_client(get_default_localnet_config("indexer"))
+
+
+@pytest.fixture(scope="module")
+def create_avm_invoker() -> typing.Callable[[ApplicationClient], AVMInvoker]:
```

Comment:
might be simpler to just make this a helper function that takes an ApplicationClient and returns an AVMInvoker, rather than a fixture.

Then we can just have client specific fixtures as required